### PR TITLE
[11.0][MIG] base_vat_optional_vies

### PR DESCRIPTION
**Pull request by RodrigoBM**
This module extends base_vat module features allowing to know if VIES
validation was passed or not.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history adapting technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0 (as this comes from other branch different than 10.0).

**Comment by RodrigoBM** [comment]
Hi @pedrobaeza,
I have already made the changes you mentioned.
Thank

**Comment by pedrobaeza** [comment]
OK, you can squash Transbot commit together using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

Now reviewers should review the PR.

File: base_vat_optional_vies/__manifest__.py
Comment (rafaelbn) [inline base_vat_optional_vies/__manifest__.py]: Migrating a module don't means being author. Only in some cases when a refactoring o something significant. In any case if a Technical review say you have done a significant work then the author should be under Tecnativa and above OCA.

File: base_vat_optional_vies/__manifest__.py
Comment (rafaelbn) [inline base_vat_optional_vies/__manifest__.py]: Change website to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

**Review by rafaelbn — CHANGES_REQUESTED** [review]

File: base_vat_optional_vies/__manifest__.py
Comment (RodrigoBM) [reply base_vat_optional_vies/__manifest__.py]: okay, I change it

File: base_vat_optional_vies/__manifest__.py
Comment (RodrigoBM) [reply base_vat_optional_vies/__manifest__.py]: okay, I change it

**Comment by RodrigoBM** [comment]
@rafaelbn  Could you please check? Please

File: base_vat_optional_vies/views/res_partner_view.xml
Comment (Tardo) [inline base_vat_optional_vies/views/res_partner_view.xml]: Add empty line

**Review by Tardo — CHANGES_REQUESTED** [review]
Thanks for the work, it would be nice if you used the new README structure.

File: base_vat_optional_vies/views/res_partner_view.xml
Comment (RodrigoBM) [reply base_vat_optional_vies/views/res_partner_view.xml]: Hi @Tardo , I have already added empty line and updated README, i hope the README is fine.

**Comment by rafaelbn** [comment]
Sorry 😢

**Comment by rafaelbn** [comment]
Sorry @RodrigoBM I miss this one. If you want to rebase I can test it with pleasure

**Review by rafaelbn — APPROVED** [review]
Thanks for the changes

**Comment by rafaelbn** [comment]
Check travis @RodrigoBM 
```

$ export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}

4.08s$ travis_install_nightly

DEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.

DEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.

ERROR: Could not find a version that satisfies the requirement pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1)) (from versions: 0.1, 0.1.1, 0.2, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.6, 0.3, 0.4, 0.5, 0.6)

ERROR: No matching distribution found for pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1))

The command "travis_install_nightly" failed and exited with 1 during .

Your build has been stopped.
```

**Comment by RodrigoBM** [comment]
@rafaelbn The error comes from Travis. This error is also in other repisitories like https://app.travis-ci.com/github/OCA/account-financial-reporting/jobs/522506817 and I don't know how to solve it.

**Comment by rafaelbn** [comment]
@RodrigoBM I understand but if nobody fix it bot don't allow to merge it 😢

**Comment by RodrigoBM** [comment]
Hi @rafaelbn, I have tried to remove travis and switch to github actions [here](https://github.com/OCA/account-financial-tools/pull/1511). But I have a problem with the account_spread_contract module, it doesn't pass the test.

**Pull request closed without merging**
